pom.xml

```diff
@@ -17,14 +17,18 @@
     <dependency>
       <groupId>org.openjfx</groupId>
       <artifactId>javafx-controls</artifactId>
-      <version>17.0.6</version>
+      <version>20</version>
     </dependency>
```

Comment:
format the code,
for however cute java 20 might feel and look, please restrict yourself to java 9
also only create the bare bones of the application, just something that will start up, a blank, the other features of the application are addressed in the other tickets